spec/services/idv/gpo_mail_spec.rb

```diff
@@ -3,76 +3,181 @@
 RSpec.describe Idv::GpoMail do
   let(:user) { create(:user) }
   let(:subject) { Idv::GpoMail.new(user) }
+  let(:max_mail_events) { 2 }
+  let(:mail_events_window_days) { 30 }
+  let(:minimum_wait_before_another_usps_letter_in_hours) { 24 }
+
+  before do
+    stub_const 'Idv::GpoMail::MAX_MAIL_EVENTS', max_mail_events
+    stub_const 'Idv::GpoMail::MAIL_EVENTS_WINDOW_DAYS', mail_events_window_days
+    stub_const 'Idv::GpoMail::MINIMUM_WAIT_BEFORE_ANOTHER_USPS_LETTER_IN_HOURS',
+               minimum_wait_before_another_usps_letter_in_hours
```

Comment:
I think it's clearer to use the pattern I've seen in the rest of the code, `allow(IdentityConfig.store).to receive(:max_mail_events)` etc. rather than introducing constants that are IdentityConfig settings in disguise.